tslint.json

```diff
@@ -11,7 +11,18 @@
         "member-access": [true, "no-public"],
         "no-empty": [true, "allow-empty-catch", "allow-empty-functions"],
         "no-string-literal": false,
-        "interface-over-type-literal": false,
+        "interface-over-type-literal": true,
+        "arrow-return-shorthand": true,
+        "no-angle-bracket-type-assertion": true,
+        "no-redundant-jsdoc": true,
+        "no-unnecessary-initializer": true,
```

Comment:
I really don't like this one (and also, using the ESLint version has caused bugs in the past). I think an initializer should be used to mean "this might be read before it's written to".

Reply:
Incidentally, LGTM will already catch a truly unnecessary initializer, based on static analysis.